src/Microsoft.ML.Featurizers/TimeSeriesImputer.cs

```diff
@@ -127,6 +137,12 @@ internal sealed class Options : TransformInputBase
 
         #region Class Enums
 
+        /// <summary>
+        /// This is the representation of which Imputation Strategy to use.
```

Comment:
Suggest the following re-word:

///&lt;summary&gt;
/// Method by which values are imputed.
/// &lt;/summary&gt;
///&lt;remarks&gt;
/// Add the extra information here ...

It would be even better if the descriptions of each value could be annotated on the value itself. I have not seen examples of this but I will check

Reply:
Let me know if you find a way to do it. I'll skip this for now until I hear back.

Reply:
We should at least put them in remarks.

Reply:
Found out that adding summaries for individual enum values *is* possible. See source example here: https://github.com/dotnet/machinelearning/blob/master/src/Microsoft.ML.AutoML/API/BinaryClassificationExperiment.cs#L45 rendered in the table here: https://docs.microsoft.com/en-us/dotnet/api/microsoft.ml.automl.binaryclassificationmetric?view=ml-dotnet-preview

Reply:
Alright, I have added in the summaries on the individual enum values.